<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doctor_add</class>
 <widget class="QDialog" name="Doctor_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Імя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="surnameLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Прізвище</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="surnameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="positionLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Посада</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cabinetLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Кабінет</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cabinetLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="phoneLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="phoneLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="loginLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Логін</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="loginLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>10</y>
     <width>28</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>44</y>
     <width>28</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>77</y>
     <width>28</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>110</y>
     <width>28</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>143</y>
     <width>28</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>320</y>
     <width>28</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Години прийому:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>298</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>327</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Початок</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>323</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Кінець</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>350</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Doctor_add</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Doctor_add</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
